docs/pages/guides/mesheryctl/working-with-mesheryctl.md

```diff
@@ -2,7 +2,7 @@
 layout: default
 title: Using mesheryctl
 description: How to use mesheryctl
-permalink: guides/mesheryctl/using-mesheryctl
+permalink: guides/mesheryctl/working-with-mesheryctl
```

Comment:
@Vidyaa123 will you confirm that this doesn't break another internal link to the previous `using-mesheryctl` permalink?

Reply:
@leecalcote, I have missed this. It does break the 'using-mesheryctl'  and 'running-system-checks using mesheryctl' internal link as well. The urls are hardcoded in toc.yml file.  I have fixed that now. and it works for me. Please review. Thanks.